src/main/java/de/mediathekview/mserver/crawler/ard/json/ArdFilmDeserializer.java

```diff
@@ -275,7 +276,7 @@ private void addUrls(final Film film, final Map<Resolution, String> videoUrls) {
       try {
         film.addUrl(
           qualitiesEntry.getKey(),
-          new FilmUrl(url, new FileSizeDeterminer(url).getFileSizeInMiB()));
+          new FilmUrl(url, crawler.determineFileSizeInKB(url)));
```

Comment:
alt in MiB, neu in KB? ist das korrekt?

Reply:
Das war jetzt die Ansage.
Alt ist MB...
Ob das für neu in KB so in Stein gemeißelt ist, wird sich zeigen